suite/contracts/contracts_flux_validate_flag_test.go

```diff
@@ -0,0 +1,155 @@
+			rac, err = s.Deployer.DeployReadAccessController(s.Wallets.Default())
+			Expect(err).ShouldNot(HaveOccurred())
+			flags, err = s.Deployer.DeployFlags(s.Wallets.Default(), rac.Address())
+			Expect(err).ShouldNot(HaveOccurred())
+			dfv, err = s.Deployer.DeployDeviationFlaggingValidator(s.Wallets.Default(), flags.Address(), big.NewInt(0))
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+		By("Deploying and funding contract", func() {
+			fmOpts := contracts.FluxAggregatorOptions{
+				PaymentAmount: big.NewInt(1),
+				Validator:     common.HexToAddress(dfv.Address()),
+				Timeout:       uint32(30),
+				MinSubValue:   big.NewInt(0),
+				MaxSubValue:   big.NewInt(1e18),
+				Decimals:      uint8(0),
+				Description:   "Hardhat Flux Aggregator",
+			}
+			fluxInstance, err = s.Deployer.DeployFluxAggregatorContract(s.Wallets.Default(), fmOpts)
+			Expect(err).ShouldNot(HaveOccurred())
+			err = fluxInstance.Fund(s.Wallets.Default(), nil, big.NewFloat(1))
+			Expect(err).ShouldNot(HaveOccurred())
+			err = fluxInstance.UpdateAvailableFunds(context.Background(), s.Wallets.Default())
+			Expect(err).ShouldNot(HaveOccurred())
+			err = s.Client.WaitForEvents()
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+		By("Setting access to flags contract", func() {
+			err = rac.AddAccess(s.Wallets.Default(), dfv.Address())
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+		By("Funding Chainlink nodes", func() {
+			nodeAddresses, err = actions.ChainlinkNodeAddresses(nodes)
+			Expect(err).ShouldNot(HaveOccurred())
+			err = actions.FundChainlinkNodes(
+				nodes,
+				s.Client,
+				s.Wallets.Default(),
+				big.NewFloat(2),
+				nil,
+			)
+			Expect(err).ShouldNot(HaveOccurred())
+		})
+		By("Setting oracle options", func() {
+			err = fluxInstance.SetOracles(s.Wallets.Default(),
+				contracts.SetOraclesOptions{
+					AddList:            nodeAddresses,
+					RemoveList:         []common.Address{},
+					AdminList:          nodeAddresses,
+					MinSubmissions:     3,
+					MaxSubmissions:     3,
+					RestartDelayRounds: 0,
+				})
+			Expect(err).ShouldNot(HaveOccurred())
+			err = s.Client.WaitForEvents()
+			Expect(err).ShouldNot(HaveOccurred())
+			oracles, err := fluxInstance.GetOracles(context.Background())
+			Expect(err).ShouldNot(HaveOccurred())
+			log.Info().Str("Oracles", strings.Join(oracles, ",")).Msg("Oracles set")
+		})
+		By("Creating flux jobs", func() {
+			for _, n := range nodes {
+				fluxSpec := &client.FluxMonitorJobSpec{
+					Name:              "flux_monitor",
+					ContractAddress:   fluxInstance.Address(),
+					PollTimerPeriod:   15 * time.Second, // min 15s
+					PollTimerDisabled: false,
+					ObservationSource: client.ObservationSourceSpec(adapter.ClusterURL() + "/variable"),
+				}
+				_, err = n.CreateJob(fluxSpec)
+				Expect(err).ShouldNot(HaveOccurred())
+			}
+		})
+	})
+
+	Describe("with Flux job", func() {
+		It("Sets a flag when value is above threshold", func() {
+			{
```

Comment:
I don't like this notation of using brackets in which I presume is organising the code. Just have it all on the same level.